config/config.exs

```diff
@@ -1,4 +1,2 @@
 use Mix.Config
 
-config :bugsnag,
```

Comment:
yes, it was intentionally, as I've moved configuration to the app module instead